docs/aws/audit/codebuildmonitoring/rules/codebuild_project_logging_enabled_remediation.mdx

```diff
@@ -1,7 +1,255 @@
+       }
+   )
+   print(response)
+   ```
+
+4. **Verify Logging Configuration**:
+   After creating or updating the project, you can verify that the logging configuration is correctly set by describing the project and checking the `logsConfig` attribute.
+   ```python
+   project_name = 'MyCodeBuildProject'
+   response = codebuild_client.batch_get_projects(names=[project_name])
+   project = response['projects'][0]
+
+   if project['logsConfig']['cloudWatchLogs']['status'] == 'ENABLED':
+       print(f"Logging is enabled for project {project_name}")
+   else:
+       print(f"Logging is not enabled for project {project_name}")
+   ```
+
+By following these steps, you can ensure that logging is enabled for your AWS CodeBuild projects using Python scripts.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console.
+2. Navigate to the AWS CodeBuild dashboard. You can do this by typing "CodeBuild" into the search bar and selecting it from the dropdown menu.
+3. In the CodeBuild dashboard, you will see a list of your CodeBuild projects. Select the project you want to check.
+4. In the project details page, scroll down to the 'Environment' section. Here, you should see a 'CloudWatch logs' field. If logging is enabled, this field will show 'ENABLED'. If it is not, it will show 'DISABLED'.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Make sure you have the necessary permissions to access the CodeBuild project.
+
+2. Once the AWS CLI is set up, you can list all the CodeBuild projects in your AWS account by running the following command:
+
+   ```
+   aws codebuild list-projects
+   ```
+
+   This command will return a list of all the CodeBuild project names.
+
+3. To get the details of a specific CodeBuild project, you can use the `batch-get-projects` command followed by the name of the project. For example, if your project name is `my-project`, you can run:
+
+   ```
+   aws codebuild batch-get-projects --names my-project
+   ```
+
+   This command will return a JSON object with all the details of the specified project.
+
+4. To check if logging is enabled for the CodeBuild project, you need to look at the `logsConfig` field in the returned JSON object. If the `cloudWatchLogs` field is set to `ENABLED`, then logging is enabled for the project. If it is set to `DISABLED`, then logging is not enabled.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) in your environment. Boto3 allows you to directly create, update, and delete AWS services from your Python scripts.
+
+```python
+pip install boto3
+```
 
+2. Import the necessary libraries and establish a session with AWS using your access keys. 
+
+```python
+import boto3
+
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>